linetools/spectra/utils.py

```diff
@@ -292,11 +304,11 @@ def rebin(spec, new_wv, do_sig=False, all=False, **kwargs):
         new_sig = None
 
     # update continuum
-    if spec.co_is_set:
-        x, y = spec._get_contpoints()
-        new_co = spec._interp_continuum(x, y, new_wv)
-    else:
-        new_co = None
+    new_co = None
+    if do_co:
+        if spec.co_is_set:
+            x, y = spec._get_contpoints()
```

Comment:
In my testing example, I instantiated an XSpectrum1D with a fresh (no continuum points in the header) FITS file, fit a continuum with fit_continuum(), and tried `spec._get_contpoints()`.  The resulting error message is appended below.  Even though `spec.co_is_set` is now `True`, there is no `spec.meta['contpoints']` (returns KeyError.

```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-33-b2e0f623c450> in <module>()
----> 1 x,y = spec._get_contpoints()

/Users/burchett/Dropbox/Code/linetools/linetools/spectra/xspectrum1d.pyc in _get_contpoints(self)
   1484             contpoints = self.meta['contpoints']
   1485         except:
-> 1486             raise RuntimeError('Meta data `contpoints` does not exist; cannot get contpoints!')
   1487         xy = np.array(contpoints)
   1488         xy = xy.transpose()

RuntimeError: Meta data `contpoints` does not exist; cannot get contpoints!
```